Source/ALSV4_CPP/Public/Character/ALSBaseCharacter.h

```diff
@@ -356,13 +356,13 @@ class ALSV4_CPP_API AALSBaseCharacter : public ACharacter
 protected:
 	/** Input */
 
-	UPROPERTY(EditDefaultsOnly, BlueprintReadWrite, Category = "Input")
```

Comment:
Is there any specific reason that those were changed? Afaik, changing those values on runtime from editor panel doesn't work correctly, the setter functions should be invoked instead of that.

Reply:
It is needed to be able to set the initial state of the character. This is used to make the different AI characters move differently in the demo scene.

Reply:
Oh, that's right. Then we can keep them like that.

We still need to prevent users from changing those variables in runtime by the way. After this PR is merged, I will look for a quick solution.